src/arm64/executor/main.c

```diff
@@ -42,9 +42,11 @@ int (*set_memory_nx)(unsigned long, int) = 0;
 long uarch_reset_rounds = UARCH_RESET_ROUNDS_DEFAULT;
 char pre_run_flush = PRE_RUN_FLUSH_DEFAULT;
 char enable_faulty_page = ENABLE_FAULTY_DEFAULT;
-char *measurement_template = (char *)&template_l1d_flush_reload; // template_l1d_prime_probe is an alternative but is currently non-functional
 char *measurement_code = NULL;
 
+char *measurement_template = (char *)&template_l1d_flush_reload;
+char *measurement_template = (char *)&template_l1d_prime_probe;
```

Comment:
This is a duplicate declaration of a global variable. I'd guess, this won't compile

Reply:
Ah! Thank you. A quick delete and failure to check the resulting code on my part. The duplicate has been removed.